python/hail/dataset.py

```diff
@@ -1259,7 +1315,9 @@ def count(self, genotypes=False):
         ``'nSamples'``, ``'nVariants'``, and ``'nGenotypes'``.
 
         :param bool genotypes: If True, include number of called
-            genotypes and genotype call rate as keys ``'nCalled'`` and ``'callRate'``, respectively.
+            genotypes and genotype call rate as keys ``'nCalled'`` and ``'callRate'``, respectively. If the genotype
+            schema does not equal :py:class:`~hail.type.TGenotype`, the definition of "called" is the genotype cell (``g``) is
```

Comment:
You're correct the cell would never be missing upon importing a VCF with generic=True. But it will be missing after filter_genotypes.